src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/JsonObject.cs

```diff
@@ -7,6 +7,7 @@
 using System.Diagnostics.CodeAnalysis;
```

Comment:
@daxian-dbw, your last commit had 2 failures in `PowerShell-CI-windows`
TabCompletion.Miscellaneous completion tests.Test completion with common parameters
```powershell
Expected a collection with size 3, but got collection with size 1 @(System.Management.Automation.CompletionResult).
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\TabCompletion\TabCompletion.Tests.ps1: line 902
902:             $res.CompletionMatches | Should -HaveCount 3
```
TabCompletion.Miscellaneous completion tests.Test completion with exact match
```powershell
Expected a collection with size 4, but got collection with size 2 @(System.Management.Automation.CompletionResult, System.Management.Automation.CompletionResult).
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\TabCompletion\TabCompletion.Tests.ps1: line 909
909:             $res.CompletionMatches | Should -HaveCount 4
```